db/schema.rb

```diff
@@ -332,6 +339,13 @@
   add_index "slack_channels", ["stage_id"], name: "index_slack_channels_on_stage_id", using: :btree
   add_index "secrets", ["id"], name: "index_secrets_on_id", unique: true, length: {"id"=>191}, using: :btree
 
+  create_table "slack_identifiers", force: :cascade do |t|
+    t.integer  "user_id",    limit: 4
+    t.text     "identifier", limit: 65535, null: false
```

Comment:
Should we define indices for these two columns, since we run a number of queries on either of them? Or is it magically done by rails?